source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -29,10 +29,13 @@ by calling the ``find()`` or ``findOne()`` method.
 
 .. tip:: Interactive Lab
    
-   You can complete a short interactive lesson that demonstrates how to
-   retrieve data by using the ``find()`` method in an in-browser coding
-   experience. To learn more, see the :ref:`node-retrieve-instruqt-lab`
-   section of this guide.
+   This page includes a short interactive lesson that demonstrates how to
+   retrieve data by using the ``find()`` method. You can complete this lab
+   directly in your browser window without installing MongoDB or a code editor.
+
+   To start the lab, click the :guilabel:`Open Interactive Tutorial` button to the
+   right of the page header. To expand the lab to a full-screen format, click the
```

Comment:
S: I think it would be sufficient and more easily understandable to say something like "at the top of the page" to help users locate the button.

```suggestion
   To start the lesson, click the :guilabel:`Open Interactive Tutorial` button at the top of the page. To expand the lesson to a full-screen format, click the
```